packages/@webex/plugin-meetings/src/breakouts/index.ts

```diff
@@ -170,6 +174,13 @@ const Breakouts = WebexPlugin.extend({
    */
   updateBreakout(params) {
     this.set(params);
+    if (params.sessionType === BREAKOUTS.SESSION_TYPES.MAIN) {
+      this.set({
+        mainGroupId: params.groupId,
+        mainSessionId: params.sessionId,
+        breakoutGroupId: params.groups && params.groups.length ? params.groups[0].id : '',
```

Comment:
question: When I looked at this value, it seems to be the same as the mainGroupId. Are you sure we actually need breakoutGroupId?

Reply:
need the breakoutGroupId (should not same with mainGroupId) when the host in bo session, and broadcast to all. I'm going to debug it again and try to get the groupId through the breakout collection instead of adding a new prop.

Reply:
I'd removed these code and get the main groupId and breakout group Id from current session and breakouts collection